ghost/webmentions/test/MentionSendingService.test.js

```diff
@@ -3,10 +3,22 @@ const assert = require('assert');
 const nock = require('nock');
 // non-standard to use externalRequest here, but this is required for the overrides in the libary, which we want to test for security reasons in combination with the package
 const externalRequest = require('../../core/core/server/lib/request-external.js');
+// non-standard to use jobservice here as well, but it's best to be jobbing off sending mentions
+const jobsService = require('../../core/core/server/services/jobs');
 const sinon = require('sinon');
 const logging = require('@tryghost/logging');
 const {createModel} = require('./utils/index.js');
 
+let jobService = {
+    async addJob(name, fn) {
```

Comment:
I would use a fake job service here, something like:
```
const jobService = {
  async addJob(name, fn) {
    return fn();
  }
};
```

Reply:
If you wanted you can also use `sinon.stub` or something to make sure that we call the `addJob` method